### PR TITLE
write a readme

### PR DESCRIPTION
**Pull request by zeke**
This PR adds a README so folks know what this repo is for and how to keep it up to date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Electron Frameworks
+
+This repository contains a collection of binary frameworks that are bundled by 
+Electron. These assets are downloaded as part of Electron's 
+[bootstrapping process](https://github.com/electron/electron/blob/master/script/update-external-binaries.py).
+
+- [DirectX SDK](https://msdn.microsoft.com/en-us/library/windows/desktop/ee663275(v=vs.85).aspx)
+- [MS Visual Studio C++ Runtime](https://msdn.microsoft.com/en-us/library/abx4dbyh.aspx)
+- [Mantle](https://github.com/Mantle)
```
Comment (joshaber) [inline README.md]: This is really https://github.com/Mantle/Mantle

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- [Reactive Cocoa](https://github.com/ReactiveCocoa/ReactiveCocoa)
+- [Squirrel](https://github.com/squirrel)
```
Comment (joshaber) [inline README.md]: And this is specifically Squirrel.Mac: http://github.com/Squirrel/Squirrel.Mac/

**Review by joshaber — CHANGES_REQUESTED** [review]
Just a couple notes 🗒

File: README.md (hunk 1 above)
Comment (zeke) [reply README.md]: gotcha. fixed.

**Review by joshaber — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Pull request merged**